app/views/shared/_recipe.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="card mx-auto my-1" style="max-width: 440px;">
+  <%= link_to recipe_path(recipe), class: "custom-link" do %>
+    <div class="card-body">
+      <h5 class="card-title"><%= recipe.name%> </h5>
+      <p class="card-text"><%= recipe.description%> </p>
+    </div>
+  <% end %>
+  <div class="text-center my-2">
+    <%= render 'shared/delete_recipe', recipe: recipe %>
```

Comment:
- [ ] Although the styling is not a requirement, your designs should meet the given wireframes, kindly add some CSS to all your pages to meet this requirement.


![recipe](https://github.com/ezraermy/Recipe-App/assets/53124548/01659a91-e766-46a2-a41a-99b694e400ff)


![recipe-design](https://github.com/ezraermy/Recipe-App/assets/53124548/8342ddd5-bb95-46f9-a6db-e0dac6968844)